Expand host variables as well

agx opened the pull request:
Instead of only allowing groups created from host group params also
allow to use host variables like subnet_name.

Closes: #5

brandonweeks commented:
:+1:

dLobatog commented on README.md:
```diff
@@ -72,20 +72,23 @@ and each of the hostgroups defines a parameters respectively:
                       "{app_param}-{tier_param}",
-                      "{app_param}"]
+                      "{app_param}",
+                      "{mysubnet}-{provision_method}"]
```
Shouldn't 'mysubnet' here be 'host_subnet' or something like that?

agx replied:
Thanks for spotting this. /subnet_name/ is the right term.

agx committed:
Expand host variables as well

Instead of only allowing groups created from host group params also
allow to use host variables like subnet_name.

Closes: #5
